client/src/routes/login/tabs/SignUp.tsx

```diff
@@ -41,7 +44,19 @@ function SignUp () {
 		e.preventDefault();
 
 		if (password === confirmPassword) {
-			// login geslaagd
+			
+			// user.setUser(
+			// 	encode(JSON.stringify({
```

Comment:
i think its a better prctice if we just write fields in the complete form instead of short